packages/core/src/tests/fixtures.ts

```diff
@@ -1,72 +1,59 @@
 /**
  * Nodes
  */
-export const rootNode = {
-  id: "canvas-ROOT",
-  data: {
-    props: {},
-    name: "Document",
-    displayName: "Document",
-    custom: {},
-    nodes: [],
-    type: { resolvedName: "Document" },
-  },
-  related: {},
-  events: { selected: false, dragged: false, hovered: false },
-  rules: {},
-};
 
-export const leafNode = {
-  id: "node-L1eGyOJ4m",
-  data: {
-    props: { childrenString: "Header 1" },
-    name: "Text",
-    displayName: "Text",
-    custom: {},
-  },
-  related: {},
-  events: { selected: false, dragged: false, hovered: false },
-  rules: {},
+export const createTestNode = (id, data, config = {}) => {
```

Comment:
oooh this is a useful one. Could you put this in utils maybe so we can also use it in other places?